cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/LocationGroupEndpointInput.java

```diff
@@ -65,11 +72,15 @@ public static final class GetOne extends EndpointInput {
         private final String groupId;
         private final String officeId;
         private final String categoryId;
+        private final String groupOfficeId;
+        private final String categoryOfficeId;
 
-        private GetOne(String categoryId, String groupId, String officeId) {
+        private GetOne(String categoryId, String groupId, String officeId, String groupOfficeId, String categoryOfficeId) {
```

Comment:
Missing the corresponding update for GetAll

Reply:
These parameters haven't been added to GetAll on the CDA side, just GetOne

Reply:
I missed this, the ts groups endpoint was updated for both getOne and getAll. loc groups need to match

Reply:
Will do